kubernetes/ama-logs.yaml

```diff
@@ -1,13 +1,13 @@
 apiVersion: v1
```

Comment:
This yaml file has still some container names with omsagent, omsagent-prometheus and omsagent-win. if you search for omsagent in this file, you can find it.

Reply:
another merge issue. fixed now